src/main/java/com/google/devtools/build/lib/profiler/chart/HtmlChartVisitor.java

```diff
@@ -313,13 +314,13 @@ private void anchor(String name) {
   /**
    * Formats the given {@link Color} to a css style color string.
    */
-  private String formatColor(Color color) {
+  public static String formatColor(Color color) {
     int r = color.getRed();
     int g = color.getGreen();
     int b = color.getBlue();
     int a = color.getAlpha();
 
-    return String.format("rgba(%d,%d,%d,%f)", r, g, b, (a / 255.0));
+    return String.format(Locale.ROOT, "rgba(%d,%d,%d,%f)", r, g, b, (a / 255.0));
```

Comment:
Why not use Locale.US here. That seems to more clearly document the purpose. Also, might want to add a comment on why we provide a locale here.